src/game/GamemodeHandler.js

```diff
@@ -20,7 +21,7 @@ class GMHandlerClass {
       dodgebot: Dodgebot,
       testGamemode: TestGamemode,
     };
-
+    /* eslint-enable object-shorthand */
```

Comment:
You should not just do an eslint disable if an error occured, resolve the error instead unless a valid argument is made

Reply:
The argument is that I followed the same naming standard as the other variables, IE camelCase. But with a abbreviation such as KO the name becomes the same as the class, an instance of KOWander is called KOWander rather than kOWonder.

Reply:
if the names are the same you do not need to specify both name and argument, {name: name} is the same as {name}. So the latter is the prefered way to do that.

Reply:
Also what is the purpose of renaming the classes? If it shows up as longer names on the UI maybe another fix would be to add a variable that holds the name of the gamemode. Seems over the top that you would have to rename the classes